mollie/api/objects/capture.py

```diff
@@ -40,14 +40,12 @@ def settlement_id(self):
     def created_at(self):
         return self._get_property("createdAt")
 
-    @property
-    def payment(self):
+    def get_payment(self):
         """Return the payment for this capture."""
-        # TODO Use the embedded payment data, if avalable.
+        # TODO Use the embedded payment data, if available.
```

Comment:
Is this available yet?

Reply:
It depends on whether the developer requested embedded payment data in the capture request: https://docs.mollie.com/reference/v2/captures-api/get-capture#embedding-of-related-resources